docs/usage/getting-started/use-cases.md

```diff
@@ -0,0 +1,135 @@
+Renovate handles both these tag scenarios and will propose updates such as from `14.17.4` to `14.7.5`, or from `14.17.4-alpine3.11` to `14.17.5-alpine3.11`.
+
+A better example of Renovate's power is when updating Docker digests.
+While humans could be reasonably expected to check and update versions like `14.17.4`, looking up image digests and updating them manually is impractical to do at scale.
+Renovate can not only keep Docker digests updated, but it can even be configured to "pin" digests from being tag-based to being tag+digest based to get immutable builds.
+
+### Internal Package Updates
+
+Companies typically have at least dozens of repositories, if not hundreds or thousands.
+In most cases, these repositories do not operate in isolation and may have upstream or downstream internal dependencies.
+In such cases, it is best practice to:
+
+- Update downstream links as soon as possible, and
+- Keep internal version use as consistent as possible
+
+Renovate is often used to achieve both the above best practices by detecting and updating internal dependencies just like external or Open Source dependencies.
+
+An example from Renovate itself is the use of submodule updating to automate the process of updating Renovate's documentation:
+
+- Renovate's main repository [`renovatebot/renovate`](https://github.com/renovatebot/renovate) contains the majority of markdown documenmtation files
+- Renovate's documentation build repository [`renovatebot/renovatebot.github.io`](https://github.com/renovatebot/renovatebot.github.io) contains a submodule link to `renovatebot/renovate`
+- Submodule updates are performed automatically whenever detected
+- After the automatic update is merged, the documentation site is rebuilt and pushed live
+
+The above use case makes use of Renovate's "automerge" feature, which allows for fully automated updates without need for manual approval, merging, or even a PR at all if desired.
+Automerge is particularly useful for internal dependencies when it's best to use the appraoch of "if it passes tests then merge it".
+
+## Advanced Configuration Use
+
+The below capabilities are common across the above use cases.
+
+### Batched Updates
+
+Although Renovate defaults to separating each dependency update into its own PR, some users prefer to batch or "group" updates together.
+For example, group all patch updates into one PR or even perhaps all non-major updates together (patches and minor updates).
+
+Renovate supports this capability using the `groupName` configuration as part of `packageRules`.
+
+### Scheduled Updates
+
+Some users prefer to limit which hours of the day or week when Renovate will raise updates.
+This may be to reduce "noise" during working hours, but also to reduce the chance of CI contention at times when developers are more likely to be waiting on tests to finish.
+
+Renovate allows users to define time ranges during which to create updates using the `schedule` field.
+
+### On-demand Updates
```

Comment:
I like this section overall, do we intend to link to the Dependency Dashboard showcase page in here eventually?

Reply:
Yes, definitely!

Reply:
Okay! 😄